### DISCUSSION TITLE
Unique visitors

### DISCUSSION THREAD

**Discussion by salpcnsi** [allinurl/goaccess#2363] [Ideas]
I understand that unique visitors are counted using IP+date+user agent. But this does not always work as I am behind a proxy so I get proxy IP.

I am an authenticated site so I do get remote user (%e). So there should be an option to count unique visitors based on %e.

I love goaccess, it is very fast and report is nicely formatted.

**Comment by allinurl** [comment]
Sounds doable. Do you have access to the XFF field?

**Reply by salpcnsi** [reply]
yes, x-forwarded-for is there but I cannot modify the logformat to include XFF at my site.

Since username is present, that maybe a good option. In the meantime I will check if XFF can replace the proxy IP.
